Automattic/wp-calypso
# People: Invite: Add site to sites-list after accepting invite.

Use flux to add the site 

__How to test__

* checkout `update/invites-add-site-right-after-accept`

__to check__
* both flows logged in and logged out flows should still work
* after accepting an invite you should be redirected to the site/post list

(I still need to make it work for logged out users)

cc @ebinnion @roccotripaldi
